assets/sass/modules/_email-factor.scss

```diff
@@ -1,12 +1,14 @@
 @mixin email-factor {
   .resend-email-infobox {
     margin-bottom: 15px;
+    .infobox-warning p span {
```

Comment:
shall we apply changes to all infobox in general (within oie scope)

Reply:
Not really, the mark-up (the span and the adjacent link) is bound to this particular resend-email-infobox, hence it should be targeted.